pkg/services/featuremgmt/registry.go

```diff
@@ -454,5 +454,11 @@ var (
 			State:       FeatureStateBeta,
 			Owner:       grafanaObservabilityLogsSquad,
 		},
+		{
+			Name:        "disableAuthenticationConfigUI",
```

Comment:
I think we should have a enable flag instead of a disable one. Then this feature would not be rolled out by default when we still don't know if this is in a stable state